source/style-guide/markup/format/substitutions.txt

```diff
@@ -0,0 +1,117 @@
+=============
+Substitutions
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Substitutions
```

Comment:
Not 100% sure, but II think these are replaced by source constants in Snooty. If we want to continue to include information that only works with old gen builds, we should identify it as such.